source/PyMaterialX/PyDefinition.cpp

```diff
@@ -49,4 +51,7 @@ void bindPyDefinition(py::module& mod)
         .def("hasLanguage", &mx::Implementation::hasLanguage)
         .def("getLanguage", &mx::Implementation::getLanguage)
         .def_readonly_static("CATEGORY", &mx::Implementation::CATEGORY);
+
+    py::class_<mx::XmlReadOptions>(mod, "XmlReadOptions")
```

Comment:
How will a Python user access the individual members of this structure?  I believe we need to make a decision about the Python API, either by exposing the underlying member variables with "def_readwrite", or by adding and binding appropriate "set" methods:

http://pybind11.readthedocs.io/en/stable/classes.html

In this case, it seems reasonable to use the "def_readwrite" approach, which mirrors the way that these variables are accessed in C++.

Also, I would recommend moving this binding to PyXmlIo.cpp, where the read methods that use this class are bound.